node/app.js

```diff
@@ -0,0 +1,27 @@
+const express      = require('express'),
+      app          = express(),
+      routes       = require('./routes/routes');
+      // bodyParser   = require('body-parser');
+   
+
+// Embedded JavaScript (allows use of js in html templates unlike vanilla markup )
+app.set("view engine", "ejs");
```

Comment:
Assuming we are gonna build a server rendered interface. Or we could go down the route of building everything using react on the client side. Should probably talk about this, thoughts @julianjuko  @JonathanYoung319 ? I think since we want a mobile app + website, the backend should be purely endpoints that return json, no need for rendering HTML, so we don't need a view engine.

Reply:
boilerplate, used for testing landing.ejs but not needed

Reply:
@lmiller1990 we gotta make the ui as responsive as possible, we gotta remember that they might not have access to the internet in which case its gonna need to be able to do everything on the app and just send data to the server. In reality the server is just to access the database most likely

Reply:
this is just specifying that we want to use ejs (which lets you embed javascript in html like this)

```html
<div>
  <%= console.log('what') %>
</div>
```

but anyway we can leave it for now since we will have a website of some kind (even if just simple landing page).